### PR TITLE
Feat consultant offer

### PR DESCRIPTION
**Pull request by BastienDmr**

### CODE REVIEW

**Review by DylanRobinaud — APPROVED** [review]

**Review by DylanRobinaud — DISMISSED** [review]

**Review by BertrandLrc — APPROVED** [review]

**Review by ITrogg — DISMISSED** [review]

File: client/src/main.jsx [hunk 1]
```diff
@@ -33,6 +35,22 @@ const router = createBrowserRouter([
...
+  {
+    path: "/consultants/",
+    element: <ConsultantLayout />,
+    children: [
+      {
+        path: ":id/offers",
+        element: <BoardConsultant />,
+        loader: async ({ params }) => {
+          const response = await connexion.get(
+            `/api/offersconsultant/${params.id}`
```
Comment (jujuck) [inline client/src/main.jsx:47]: non juste `api/offers`

File: server/app/controllers/offerActions.js [hunk 2]
```diff
@@ -12,6 +12,23 @@ const browse = async (req, res, next) => {
     next(err);
   }
 };
+const browseOffersByConsultant = async (req, res, next) => {
```
Comment (jujuck) [inline server/app/controllers/offerActions.js:15]: browseByConsultant

File: server/app/controllers/offerActions.js [hunk 3]
```diff
@@ -12,6 +12,23 @@ const browse = async (req, res, next) => {
...
+  try {
+    // Fetch all offers from the database
+    const offersByConsultant = await tables.offer.readAllConsultantOffer(
```
Comment (jujuck) [inline server/app/controllers/offerActions.js:18]: readAllByConsultant(1)

File: client/src/pages/backOffice/ConsultantLayout/BoardConsultant.jsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { useLoaderData } from "react-router-dom";
+
+import BoardList from "../../../components/backOffice/boardComponent/BoardList";
+
+function BoardConsultant() {
+  const offers = useLoaderData();
+  const formatOffers = offers.map((offer) => ({
```
Comment (jujuck) [inline client/src/pages/backOffice/ConsultantLayout/BoardConsultant.jsx:7]: A faire dans le controller

**Review by jujuck — CHANGES_REQUESTED** [review]

**Review by lub04 — APPROVED** [review]

**Review by OrestiHoxhallari — APPROVED** [review]

**Review by jujuck — APPROVED** [review]

**Pull request merged**